### DISCUSSION TITLE
Is analyzing address transaction data supported?

### DISCUSSION THREAD

**Discussion by zitup** [MetaMask/snaps#1744] [Ideas]
Snap is great and opens up a lot of possibilities. I’m curious whether data for analyzing addresses is now supported, such as users’ token costs, yields, etc.

Is this kind of snap application supported?

**Comment by Montoya** [comment]
With the `eth_accounts` dynamic permission you can do `eth_requestAccounts` from a Snap (with the `endowment:ethereum-provider`) and then do whatever analysis you need to analyze that address and provide information. I think "Assets Risk Detection" by GoPlus already does this. 

https://docs.metamask.io/snaps/reference/permissions/#eth_accounts

**Reply by zitup** [reply]
There are currently 32 snap apps. When and how will the snaps developed in the future be released?

**Reply by Montoya** [reply]
Hi @zitup, new Snaps are added to our allowlist here as they complete the process: https://github.com/MetaMask/snaps-registry

We actually added another one yesterday. Once a Snap is added to the allowlist, the directory at https://snaps.metamask.io is updated and it will appear there.

**Reply by zitup** [reply]
Cool, if we complete the development, where will we submit it to you?

**Reply by Montoya** [reply]
Please email me: Christian.Montoya@consensys.net and I can assist you with the allowlist process
